test/Translate/openqasm2_adj_rotations.cpp

```diff
@@ -49,13 +51,13 @@ int main() {
 
 // CHECK:  qreg var0[6];
 // CHECK:  x var0[5];
-// CHECK:  ch var0[4], var0[5];
+// CHECK:  h var0[4], var0[5];
```

Comment:
this is not the same as ch; this will instead lead to a broadcasting. 
We have two options:
- translate ch into a singly-controlled rotation (seems like the best path)
- see if AWS supports ctrl(3) @ ... for multi-controlled gates.